src/VisualStudio/CSharp/Repl/Commands.vsct

```diff
@@ -23,25 +23,9 @@
           <ButtonText>C# Interactive</ButtonText>
         </Strings>
       </Button>
-
-      <Button guid="guidCSharpInteractiveCommandSet" id="cmdidResetInteractiveFromProject" priority="0x0100" type="Button">
-        <Parent guid="guidCSharpInteractiveCommandSet" id="IDG_INTERACTIVE_PROJECT"/>
-        <CommandFlag>DynamicVisibility</CommandFlag>
-        <CommandFlag>DefaultInvisible</CommandFlag>
-        <Strings>
-          <ButtonText>Reset C# Interactive from Project</ButtonText>
-          <CanonicalName>ResetC#InteractiveFromProject</CanonicalName>
-          <LocCanonicalName>ResetC#InteractiveFromProject</LocCanonicalName>
-        </Strings>
-      </Button>
     </Buttons>
   </Commands>
   
-  <VisibilityConstraints>
-    <VisibilityItem context="guidCSProjectContext" guid="guidCSharpInteractiveCommandSet" id="cmdidResetInteractiveFromProject" />
-    <VisibilityItem context="guidVBProjectContext" guid="guidCSharpInteractiveCommandSet" id="cmdidResetInteractiveFromProject" />
-  </VisibilityConstraints>
-  
```

Comment:
The visibility constraints have been completely replaced by the BeforeQueryStatus in the `AbstractResetInteractiveMenuCommand.cs`.
Constraints were used as a mechanism to defer loading of the reset command. However, it was important to actually call the QueryStatus directly in order to restrict the visibility of command further. Since QueryStatus is called directly constraints are no longer needed.